app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    redirect_to user_posts_path(id: @comment.post_id, author_id: @comment.author_id) if @comment.save
+  end
```

Comment:
- [ ] A comment should be created on behalf of the currently logged in user (current_user) of the app

Reply:
Hi @mikethreels , I appreciate the review. However, I am not satisfied with the reason for the required changes because the code is working according to the requirements.